app/models/organization.rb

```diff
@@ -0,0 +1,6 @@
+class Organization < ActiveRecord::Base
```

Comment:
In the interface, you're calling these classrooms. I'd suggest just calling everything classrooms. Organization is a thing on GitHub, `Classroom` is a thing in this app.